internal/db/applications.go

```diff
@@ -23,6 +23,17 @@ const (
 			FROM applicationversion av
 			WHERE av.application_id = a.id
 		), array[]::record[]) as versions `
+
+	applicationWithLicensedVersionsOutputExpr = applicationOutputExpr + `,
```

Comment:
Maybe I'm missing something, but I think this doesn't check the application license owner